src/ClusterBootstrap/cloud_init_deploy.py

```diff
@@ -918,11 +918,6 @@ def gen_pass_secret_script(config):
         "./template/secret/pass_secret.sh.template", "deploy/scripts/pass_secret.sh", config)
 
 
-def render_repairmanager(config):
```

Comment:
@YinYangOfDao can you help check the code to see if repairmanager can still be deployed correctly in cloud init pipeline? repairmanager currently only uses ConfigMap for storing configs.
Manual deployment commands:
```
./ctl.py docker push repairmanager
./ctl.py svc start repairmanager
```